docs/2.8.0/docs/canton/usermanual/installation.rst

```diff
@@ -92,11 +92,11 @@ Selecting your Storage Layer
 ----------------------------
 
 In order to run any kind of node, you need to decide how and if you want to persist the
-data. You currently have three choices: don't persist and just use in-memory stores which will be deleted if you restart
+data. You currently have three choices: Don't persist and just use in-memory stores which will be deleted if you restart
 your node or persist using ``Postgres`` or ``Oracle`` databases.
```

Comment:
```suggestion
data. You can choose not to persist data and instead use in-memory stores that are deleted on node restart,
or you can persist data using ``Postgres`` or ``Oracle`` databases.
```

Reply:
Note: I know you were just capitalizing "Don't," but I took the opportunity to clarify the wording overall.

Or, if you want to keep the text as is, revert the capitalization of "Don't." Our style is to use lowercase after a colon: https://developers.google.com/style/capitalization#capitalization-and-colons